docs/docs/llms/MLflow Tracing/introduction.mdx

```diff
@@ -0,0 +1,1146 @@
+    import openai
+    import mlflow
+
+    # Calling the autolog API will enable trace logging by default.
+    mlflow.openai.autolog()
+
+    mlflow.set_experiment("OpenAI")
+
+    openai_client = openai.OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
+
+    messages = [
+        {
+            "role": "user",
+            "content": "How can I improve my resting metabolic rate most effectively?",
+        }
+    ]
+
+    response = openai_client.chat.completions.create(
+        model="gpt-4o",
+        messages=messages,
+        temperature=0.99,
+    )
+
+    print(response)
+    ```
+
+    The logged trace, associated with the `OpenAI` experiment, can be seen in the MLflow UI, as shown below:
+
+    ![OpenAI Tracing](/images/llms/tracing/openai-tracing.png)
+
+  </TabItem>
+  <TabItem value="swarm" label="Swarm">
+    ### OpenAI Swarm Automatic Tracing
+
+    The MLflow OpenAI flavor supports automatic tracing for [Swarm](https://github.com/openai/swarm), a multi-agent orchestration
+    framework from OpenAI. To enable tracing for **Swarm**, just call <APILink fn="mlflow.openai.autolog" />
+    before running your multi-agent interactions. MLflow will trace all LLM interactions, tool calls, and agent operations automatically.
+
+    ```python
+    import mlflow
+
+    mlflow.openai.autolog()
+    ```
+
+    For example, the code below will run the simplest example of multi-agent interaction using OpenAI Swarm.
+
+    ```python
+    import mlflow
+    from swarm import Swarm, Agent
+
+    # Calling the autolog API will enable trace logging by default.
+    mlflow.openai.autolog()
+
+    mlflow.set_experiment("OpenAI Swarm")
+
+    client = Swarm()
+
+
+    def transfer_to_agent_b():
+        return agent_b
+
+
+    agent_a = Agent(
+        name="Agent A",
+        instructions="You are a helpful agent.",
+        functions=[transfer_to_agent_b],
+    )
+
+    agent_b = Agent(
+        name="Agent B",
+        instructions="Only speak in Haikus.",
+    )
+
+    response = client.run(
+        agent=agent_a,
+        messages=[{"role": "user", "content": "I want to talk to agent B."}],
+    )
+    print(response)
+    ```
+
+    The logged trace, associated with the `OpenAI Swarm` experiment, can be seen in the MLflow UI, as shown below:
+
+    ![OpenAI Swarm Tracing](/images/llms/tracing/openai-swarm-tracing.png)
+
+  </TabItem>
+  <TabItem value="llamaindex" label="LlamaIndex">
+    ### LlamaIndex Automatic Tracing
+
+    The MLflow LlamaIndex flavor's autologging feature has a direct integration with MLflow tracing. When LlamaIndex autologging is enabled with <APILink fn="mlflow.llama_index.autolog" />, invocation of components
+    such as LLMs, agents, and query/chat engines will automatically record generated traces during interactive development.
+
+    ```python
+    import mlflow
+
+    mlflow.llama_index.autolog()
+    ```
+
+    To see the full example of tracing LlamaIndex, please visit [LLamaIndex Tracing documentation](_).
```

Comment:
For the links that don't exist yet, we can still fill in the link, because we know what path it will eventually have. Then when we add those pages, we don't need to come back to fill in these blanks.

```suggestion
    To see the full example of tracing LlamaIndex, please visit [LLamaIndex Tracing documentation](../llama-index/#enable-tracing).
```